tools/ci.psm1

```diff
@@ -28,41 +28,7 @@ if($PSVersionTable.PSEdition -eq 'Desktop' -or $isWindows)
 # or is a pushed tag
 Function Test-DailyBuild
```

Comment:
There are some issues that occur when daily is true.  We need to fix that.  Linux and MacOS don't fail.  We need to review each case and make sure we do the right thing.

Reply:
Will do